src/main/ml-config/security/roles/data-explorer-wizard-role.json

```diff
@@ -6,5 +6,9 @@
     "privilege-name" : "data-explorer-access",
     "action" : "http://marklogic.com/ps/data-explorer-access",
     "kind" : "execute"
+  },{
```

Comment:
One thing to consider for future changes - I've found it's usually worth adding a comment for why a privilege is granted to a role. Can be difficult to figure out later otherwise.